dashboard/test/ui/features/sharepage.feature

```diff
@@ -54,9 +54,9 @@ Scenario: Share and save an artist level to the project gallery
   And element ".modal-body" is not visible
 
   Then I am on "http://studio.code.org/projects/"
-  And I wait until element "table.projects" is visible
-  And the project list contains 1 entry
-  And the project at index 0 is named "Artist Project"
+  And I wait until element ".ui-personal-projects-table" is visible
+  And the project table contains 1 row
+  And there is a project in the table named "Artist Project"
```

Comment:
I'm okay sacrificing testing the order of the projects in the table here since I'm using Enzyme tests for the `PersonalProjectsTable` to confirm that they are ordered correctly by recency.